File: pingpong.rb
puts "Please, enter a number."

num = gets.to_i
i = 0

puts "Thanks, let's play ping-pong."

while i < num do

  i += 1
  if (i % 3 == 0) && (i % 5 == 0)
    puts "Ping-Pong"
  elsif (i % 3 == 0)
    puts "Ping"
  elsif (i % 5 == 0)
    puts "Pong"
  else
  puts (i)
  end

end

File: Prework/PingPong.java
import java.io.Console;

public class PingPong {
  public static void main(String[] args) {
    Console myConsole = System.console();

    System.out.println("Please, enter a number.");
    String numberString = myConsole.readLine();
    Integer number = Integer.parseInt(numberString);
    System.out.println("Okay, let's play Ping-Pong.");

    pingPong(number);
  }
  public static void pingPong(Integer number) {

    for (Integer i = 0 ; i <= number ; i++ ) {
      if (( i % 3 == 0) && ( i % 5 == 0)){
        System.out.println("Ping-Pong");
      } else if ( i % 3 == 0) {
        System.out.println("Ping");
      } else if ( i % 5 == 0) {
        System.out.println("Pong");
      } else {
        System.out.println(i);
      }
    }

  }
}
